curriculum/challenges/english/01-responsive-web-design/css-piano/part-003.md

```diff
@@ -0,0 +1,62 @@
+---
+id: 612e7732a7edbf1d14fb6648
+title: Part 3
+challengeType: 0
+dashedName: part-3
+---
+
+# --description--
+
+Oops! You forgot the `doctype` declaration. Above your other elements, add `<!DOCTYPE html>` so the browser knows what type of document to expect.
+
+# --hints--
+
+Your code should contain the `DOCTYPE` reference.
+
+```js
+assert(code.match(/<!DOCTYPE/gi));
+```
+
+You should include a space after the `DOCTYPE` reference.
+
+```js
+assert(code.match(/<!DOCTYPE\s+/gi));
+```
+
+You should define the document type to be `html`.
+
+```js
+assert(code.match(/html/gi));
```

Comment:
This is going to match, regardless, because of the `html` tags. Either remove these tests, OR keep extending the RegEx. Example: 
1) Test for `!DOCTYPE`
2) Test for `!DOCTYPE html`
3) Test for `!DOCTYPE html>`